lib/db/schema/patch-033-034.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  code VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, code)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createUnblockCode_0` (
+    IN inUid BINARY(16),
+    IN inCode VARCHAR(32),
+    IN inCreatedAt BIGINT SIGNED
+)
+BEGIN
+    INSERT INTO unblockCodes(
```

Comment:
I realize the chances are very very small since createdAt comes from Date.now above, but what happens if two of the same rows are attempted to be added?